features/steps/permissions_component.go

```diff
@@ -0,0 +1,154 @@
+package steps
+
+import (
+	"context"
+	"fmt"
+	componenttest "github.com/ONSdigital/dp-component-test"
+	"github.com/ONSdigital/dp-healthcheck/healthcheck"
+	dpMongoDriver "github.com/ONSdigital/dp-mongodb/v2/mongodb"
+	"github.com/ONSdigital/dp-permissions-api/api"
+	"github.com/ONSdigital/dp-permissions-api/config"
+	"github.com/ONSdigital/dp-permissions-api/mongo"
+	"github.com/ONSdigital/dp-permissions-api/service"
+	serviceMock "github.com/ONSdigital/dp-permissions-api/service/mock"
+	"github.com/benweissmann/memongo"
+	"github.com/cucumber/godog"
+	"github.com/gofrs/uuid"
+	"go.mongodb.org/mongo-driver/bson"
+	"net/http"
+)
+
+type PermissionsComponent struct {
```

Comment:
Godoc on public types.

Reply:
This type is used only for component tests, Do we need to add goDoc even for types used in test?

Reply:
I didn't notice this was a test when I wrote this initially. Not as important but I guess a comment would be helpful to future maintainers.